tests/suites/os/tests/config-json/index.js

```diff
@@ -118,8 +118,18 @@ module.exports = {
 						this.context.get().link,
 					);
 
-				// Rebooting the DUT
-				await this.context.get().worker.rebootDut(this.context.get().link);
+				// Wait for new NTP server to be active
+				test.comment(`Waiting for balena-ntp-config service to be active...`);
+				await this.context.get().utils.waitUntil(async () => {
+					return (
+						(await this.context
+							.get()
+							.worker.executeCommandInHostOS(
+								`systemctl is-active balena-ntp-config.service`,
+								this.context.get().link,
+							)) === 'active'
+					);
+				}, false);
```

Comment:
In case this fails it's actually better to reduce the number of attempts here to 5.
```suggestion
				}, false, 5, 30000);
```
This way we don't wait 40mins because the timeouts/retries of `executeCommandInHostOS` and `waitUntil` will stack!

I think we plan to change the default behaviour upstream, but this helps for now.